include/plus4.h

```diff
@@ -59,6 +59,14 @@
 extern void plus4_stdjoy_joy[];   /* Referred to by joy_static_stddrv[] */
 extern void plus4_stdser_ser[];
 
+void fast (void);
+/* Switch the CPU into double clock mode. */
+
+void slow (void);
+/* Switch the CPU into single clock mode. */
+
+unsigned char isfast (void);
+/* Returns 1 if the CPU is in double clock mode. */
```

Comment:
Those declarations belong in `<cbm264.h>`.

Reply:
@greg-king5: Are you willing to adjust this (either via direct push to master or via pull rquest) ?

Reply:
> Are you willing to adjust this (either via direct push to master, or via pull request)?

Yes, very easy to move.

Reply:
:-)